app/iiif/tools.py

```diff
@@ -29,6 +29,7 @@ def get_info_from_iiif_url(iiif_url):
     try:
         relevant_url_part = iiif_url.split('edepot:')[1].split('/')[0]
         stadsdeel, dossier, document_and_file = relevant_url_part.split('-')
+        stadsdeel = stadsdeel[1]  # remove the S that prepends the stadsdeel value
```

Comment:
you could do a quick check/assert here to make sure stadsdeel[0] is indeed equal to 'S'.